src/clips-specs/rcll-central/domain.clp

```diff
@@ -303,7 +303,7 @@
 		(if (and (= (length$ ?r-active) 2) (eq (nth$ 2 ?r-active) active)
 		                                   (eq ?cf:value TRUE))
 		  then
-		    (printout error ?r-active crlf)
+		    (printout t ?r-active crlf)
```

Comment:
reduced visibility, I always relied on seeing 3 red lines when starting up

Reply:
too bad for you!